### PR TITLE
(maint) Revendor DSC resources to 9e8fb80

### PR DESCRIPTION
**Pull request by Iristyle**
(maint) Revendor DSC resources to 9e8fb80

  Long commit: 9e8fb80252483a13992e1d71b65ded99266a4f9b

 - Rerun `DSC_REF=9e8fb80 bundle exec rake dsc:build` to produce
   Puppet wrappers / re-vendor newer DSC resources.

 Adds types:

 # xActiveDirectory
 dsc_xaddomaindefaultpasswordpolicy

 # xCertificate
 dsc_xcertificateimport

 # xComputerManagement
 dsc_xofflinedomainjoin

 # xDnsServer
 dsc_xdnsserverforwarder

 # xFailOverCluster
 dsc_xclusterdisk
 dsc_xclusternetwork

 # xNetworking
 dsc_xhostsfile

 Removes types:
  
 # xCertificateServices
 dsc_xadcscertificationauthority
 dsc_xadcsonlineresponder
 dsc_xadcswebenrollment

### CODE REVIEW

**Comment by Iristyle** [comment]
@puppetlabs/windows this has to be merged prior to #202

File: build/dsc.rake [hunk 1]
```diff
@@ -57,8 +57,8 @@ eod
         puts "Downloading and Importing #{item_name}"
         cmd = "git clone #{dsc_repo} #{dsc_resources_path_tmp} && " +
           "cd #{dsc_resources_path_tmp} && "
-        cmd += "git checkout #{ENV['DSC_REF']} &&" if ENV['DSC_REF']
-        cmd += "git submodule update --init --recursive"
+        cmd += "git checkout #{ENV['DSC_REF']} && " if ENV['DSC_REF']
+        cmd += "git submodule update --init"
```
Comment (ferventcoder) [inline build/dsc.rake]: No more submodule of a submodule.

**Comment by ferventcoder** [comment]
Now that this is done - next @Iristyle

File: build/dsc.rake (hunk 1 above)
Comment (Iristyle) [inline build/dsc.rake]: Correct - OK for now given that didn't have a bearing on the generated types.

We should verify for sure though that we didn't exclude anything needed at runtime that wouldn't show up as different in type generation.

**Pull request merged**